zookeeper/source/zk_dn_client.cc

```diff
@@ -94,52 +94,12 @@ namespace zkclient{
 		return created_correctly;
 	}
 
-	bool ZkClientDn::blockDeleted(uint64_t uuid) {
```

Comment:
Why get rid of this? We still need to update block locations, block set to show that this datanode no longer has the block.